tests/org.codechecker.eclipse.rcp.shared/.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
wouldn't be better to use one gitignore file in the root of the repository, to see every ignored file or directory there?

Reply:
Thanks for the review @gyorb! Creating a new project through the wizard will create this file. I'll merge this into the root .gitignore.